opendut-lea/src/components/auth.rs

```diff
@@ -58,6 +55,11 @@ pub fn LeaAuthenticated(
                     />
                 </Transition>
             }
+
+        }
+        _ => {
+            tracing::warn!("Warning: Authentication disabled. No issuer URL was provided by the config - not able to determine the authenticated user.");
```

Comment:
Add different warning log in case authentication config is present but not authenticated.